### PR TITLE
remove `lazy_static` from `build.rs`

### PR DESCRIPTION
**Pull request by lzutao**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -79,7 +79,7 @@ assert_cmd = "1.0.1"
 [build-dependencies]
 clap = { version = "2.33", optional = true }
 liquid = { version = "0.20", optional = true }
-lazy_static = { version = "1.4", optional = true }
 
 [profile.release]
 lto = true
+codegen-units = 1
```
Comment (sharkdp) [inline Cargo.toml:85]: Could you please explain what this does?

File: appveyor.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ environment:
 
 install:
   - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
-  - rustup-init.exe -y --default-host %TARGET%
+  - rustup-init.exe -y --default-host %TARGET% --profile minimal
```
Comment (sharkdp) [inline appveyor.yml:11]: ... and this?

**Comment by sharkdp** [comment]
Looks great, thank you!

**Comment by sharkdp** [comment]
rebasing onto master to fix the build (which I accidentally broke for a minute)

File: Cargo.toml (hunk 1 above)
Comment (lzutao) [reply Cargo.toml:85]: https://doc.rust-lang.org/nightly/rustc/codegen-options/index.html#codegen-units
explain it better than me.

File: appveyor.yml (hunk 2 above)
Comment (lzutao) [reply appveyor.yml:11]: This skips installing rust-docs which takes minutes to installing on Windows: https://github.com/rust-lang/rustup#profiles

**Pull request merged**